api/server/service/jct/api.coffee

```diff
@@ -52,11 +52,14 @@ funders will be a list given to us by JCT detailing their particular requirement
 # (for convenience the settings have initially been set up to only run import on dev as well, to make the most 
 # of the dev machine and minimise any potential memory or CPU intense work 
 index_name = API.settings.es.index ? 'jct'
-jct_journal = new API.collection {index:index_name, type:"journal"}
 jct_agreement = new API.collection {index:index_name, type:"agreement"}
 jct_compliance = new API.collection {index:index_name, type:"compliance"}
 jct_unknown = new API.collection {index:index_name, type:"unknown"}
 
+# Journals are held in an alias of the name `[index_name]_journal]`.  Within that
+# index is a single type `journal` which contains the relevant data
+jct_journal = new API.collection {index:index_name + "_journals", type:"journals"}
```

Comment:
Comment mismatch - journal or journals?

Reply:
Yeah, it probably should be singular, which would match with all the other index names, I'll put that change in